wled00/wled.h

```diff
@@ -10,6 +10,19 @@
 // version code in format yymmddb (b = daily build)
 #define VERSION 2310130
 
+// You can define custom product info from build flags.
+// This is useful to allow API consumer to identify what type of WLED version
+// they are interacting with.
+// Use like this:
+// -D WLED_BRAND="F(\"Custom Brand\")"
+// -D WLED_PRODUCT_NAME="F(\"Custom Product\")"
+#ifndef WLED_BRAND
+  #define WLED_BRAND F("WLED")
```

Comment:
I would prefer F() macro was not used in #define but rather in JSON assignment.

Reply:
Done!